src/OrleansRuntime/Silo/Silo.cs

```diff
@@ -588,10 +588,6 @@ private void DoStart()
                 }
                 if (logger.IsVerbose) { logger.Verbose("Message gateway service started successfully."); }
 
-                scheduler.QueueTask(clientRegistrar.Start, clientRegistrar.SchedulingContext)
-                    .WaitWithThrow(initTimeout);
```

Comment:
The odd thing is that this stared happened for a user, but it's not like he explicitly remove the gateway from the silo. Anyway,  I'll merge this as it makes sense regardless